qemu/tests/cfg/apicv_test.cfg

```diff
@@ -0,0 +1,20 @@
+- apicv_test:
+    virt_test_type = qemu
+    type = apicv_test
+    start_vm = no
+    kill_vm_on_error = yes
+    only HostCpuVendor.intel
+    only virtio_net
+    netperf_server_link = netperf-2.7.1.tar.bz2
+    server_path = /var/tmp/
+    test_option = "-n 4 -l 60 -t TCP_STREAM -- -m 580"
+    repeat_times = 100
+    module_name = kvm_intel
+    mod_param = enable_apicv
+    Linux:
+        netperf_client_link = ${netperf_server_link}
+        client_path = ${server_path}
+    Windows:
+        netperf_client_link = "netperf.exe"
+        client_path = "c:\\"
+   
```

Comment:
Removed, any more comments?

Reply:
Miss the pre comment ?  `Could you explain more why not make vhost and ioevenfd configurable in test case ?`

Reply:
Oh, I just saw this. The reasons are, 
1. vhost=off in case cfg would be overwritten by 'vhost = "vhost=on"' in host-os-setting.cfg.
2. If configure vhost and ioeventfd in py, the code structure would be a mass, not as simple as current.
3. Current case takes more than 3 hours to finish. If put all 3 scenarios in one case, that's 9 hours for just one case, which seems not right.

Reply:
> Oh, I just saw this. The reasons are,
> 
>     1. vhost=off in case cfg would be overwritten by 'vhost = "vhost=on"' in host-os-setting.cfg.
> 

Eh~  the overwritten issue again, and use nic name as suffix can fix it.
The default nic name is nic1 in the framework:  `vhost_nic1 = ***`

>     2. If configure vhost and ioeventfd in py, the code structure would be a mass, not as simple as current.
> 
>     3. Current case takes more than 3 hours to finish. If put all 3 scenarios together, that's 9 hours for just one case, which seems not right.

For 2 and 3,  I meant configure in the cfg, use three sub cases to cover the 3 scenario, same with the configuration you added in kar.

Reply:
From my view, it is better to configure different test scenario in test case than test loop if no particularly blocker ~